### PR TITLE
Add authors

### LINKED ISSUE #21: Set up authors?
We could leave authors as "the astropy developers", or should it go to be more erfa specific? From `git shortlog`, I'd infer that the contributors are (alphabetical):
- Antonio Valentino
- Erik Tollerud
- Julien Woillez
- Marten van Kerkwijk
- Michael Droettboom
- Thomas Robitaille

And maybe
- Erik M. Bray(?)
- Stuart Littlefair(?)

### PR DESCRIPTION
**Pull request by mhvk**
fixes #21 

I ended up giving brief, one-sentence descriptions of the various contributions - since those should be checked, requesting review from all (but mdboom, who I probably should not bother with this). cc @jwoillez, who doesn't come up for "reviewers"

### CODE REVIEW

File: AUTHORS.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+* Julien Woillez, who created the first Jinja2_ template, for wrappers
+  based on an array iterator.
+* Erik Tollerud, who helped with the initial wrapping and with integration.
+* Michael Droettboom, who helped with the initial wrapping.
+* Marten van Kerkwijk, who changed the wrappers to use Numpy_
+  universal functions.
+* Antonio Valentino, who created PyERFA from the implementation in
+  Astropy_.
+* Thomas Robitaille, who helped make most steps possible.
```
Comment (astrofrog) [inline AUTHORS.rst:15]: Perhaps an overstatement of my contributions ;) Maybe you can just say something like 'who helped with the transition to a standalone package' or something like that

Comment (mhvk) [reply AUTHORS.rst:15]: You may have forgotten, but you helped with earlier steps too! I'll rephrase "helped with many of the steps".

**Comment by gellule** [comment]
All fine from my side. Good to see pyerfa becoming its own package.

**Comment by jwoillez** [comment]
All fine with me. Good to see pyerfa becoming its own package.

**Comment by mhvk** [comment]
Coming back to pyerfa now after a bit of an hiatus -- other things do crop up!  This was approved and can always be changed, so let me go ahead and merge.

**Pull request merged**